VsCodeExtension/Server/Core.fs

```diff
@@ -126,7 +129,9 @@ let SemanticTokenModifiers =
 let initialize(input: Input<InitializeParams>): Async<Output<InitializeResult>> = async {
     if (not MSBuildLocator.IsRegistered) then MSBuildLocator.RegisterDefaults() |> ignore
     let workspace = MSBuildWorkspace.Create()
-    let! _ = (workspace.OpenSolutionAsync(input.Parameters.RootPath.Value) |> Async.AwaitTask)
+    let slnPaths = System.IO.Directory.GetFiles(input.Parameters.RootPath.Value, "*.sln")
```

Comment:
Can we name this `solutionPaths`? Makes sense to me to not use contractions in names.